gpu-jobs.md

```diff
@@ -62,46 +74,60 @@ capabilities that are available to any CHTC user (as of 2/7/2020):
     <td>CentOS 7</td>
     <td>yes</td>
   </tr>
-  <tr>
```

Comment:
The GPU Lab servers were renamed.  They are no longer gpu2002 - gpu2005.  The current names are  gpulab2000 - gpulab2003.